CHANGELOG.md

```diff
@@ -1,3 +1,43 @@
+# 11.0.0-rc.2 "vicuna-valley" (2020-11-10)
```

Comment:
nit: looked up what a vicuna is and it appears to not be a material

Reply:
🦙

Reply:
TIL: a vicuna is an animal and also the name of the material made of its fur 🙂